### ISSUE TITLE
Cannot disable sound

### ISSUE THREAD

**Issue by Xiarno** [Corosauce/WATUT#5]
I disabled the sound in the config but I still ear every sound that are supposed to be disabled.
Using latest mod version, 1.20.1-1.0.11

```
#General mod settings
[general]
	#Sends relative mouse position and clicking
	sendMouseInfo = true
	#Sends a calculated rate of typing. If off, uses a default value on server
	sendTypingSpeed = true
	#Sends when you open a gui, and what gui
	sendActiveGui = true
	#Sends when you go idle / return
	sendIdleState = true
	#-
	showIdleStatesInPlayerList = true
	#-
	showIdleStatesInPlayerAboveHead = true
	#-
	showPlayerAnimations = true
	#-
	playScreenOpenSounds = false
	#-
	playMouseClickSounds = false

```

**Comment by Corosauce** [comment]
Can't reproduce, make sure coroutil is updated to 1.3.3, previous versions had a rare issue where configs werent loading correctly, hoping that is the issue. 

This config is client side only so make sure to have it set on the clients.

**Comment by Xiarno** [comment]
> Can't reproduce, make sure coroutil is updated to 1.3.3, previous versions had a rare issue where configs werent loading correctly, hoping that is the issue.
> 
> This config is client side only so make sure to have it set on the clients.

Coroutils is indeed updated to 1.3.3 and the config was changed clientside.

**Comment by Xiarno** [comment]
If that helps, I'm on the All The Mods 9 modpack.

**Comment by Corosauce** [comment]
Looks like I jumped to conclusions and messed up the fix in coroutil for 1.3.3, sorry about that, I'm aware of it now, revert to coroutil 1.3.0 for now to make it less problematic. Will be fixing this ASAP.

**Comment by Corosauce** [comment]
Fixed in coroutil 1.3.4 https://www.curseforge.com/minecraft/mc-mods/coroutil/files/4974181
